docker-compose.yml

```diff
@@ -23,6 +23,9 @@ services:
     container_name: activepieces-frontend
     restart: unless-stopped
     depends_on: [ backend ]
+    environment:
+      - AP_BACKEND_URL=${AP_BACKEND_URL}
+      - NGINX_ENVSUBST_OUTPUT_DIR=/etc/nginx
```

Comment:
This environment variable doesn't seem to be used anywhere, no?

Reply:
It's used inside the nginx config